receiver/singletonreceivercreator/factory.go

```diff
@@ -0,0 +1,49 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
You use these headers inconsistently. If you add any copyright claims, make them proper. If any of the files have been copied from other Otel collector repos, make that clear and add the source.